README.md

```diff
@@ -44,12 +44,12 @@ The library adheres to the interfaces in the [Interledger Protocol Core](https:/
 
 ## TODO
 
-    [ ] Fix Checkstyle issues
-    
+    [X] Fix Checkstyle issues
+    [ ] Update according to changes in dependencies
 
-	## Contributors
+## Contributors
 
-Any contribution is very much appreciated! [![gitter][gitter-image]][gitter-url]
```

Comment:
Yeah, it was appearing as plain text. I don´t know how to make the Gitter button show properly (I tried copying and pasting from other projects and it still wouldn't show) so I  just removed it, if any of you could do it it'd be very much appreciated.